### ISSUE TITLE
Button event not propagating to ViewModel inside <ak-grid><ak-template>

### ISSUE THREAD

**Issue by brandonseydel** [aurelia-ui-toolkits/aurelia-kendoui-bridge#682]
I have a simple view with a local data source that starts off empty and one object is added with the ability to remove.  The event is not firing and i tried wrapping in a div also.  This is inside of a form sith a submit.trigger so i set the type to button.  I moved outside of the form also still with no luck.  What is going on here.
```html
                    <ak-grid id="nodeParams"
                             k-data-source.bind="dataSource"
                             k-pageable.bind="{ input: true, numeric: false}"
                             k-filterable.bind="true"
                             k-sortable.bind="true"
                             k-scrollable.bind="true"
                             k-selectable.bind="false"
                             k-auto-bind="false"
                             k-widget.bind="grid"
                             k-editable="popup"
                             k-navigatable.bind="true">
                        <ak-col k-field="sequenceNbr" k-title="Seq"></ak-col>
                        <ak-col k-field="parmEqualOrRangeFrom" k-title="Parm Equals / From"></ak-col>
                        <ak-col k-field="parmRangeThru" k-title="Parm Thru"></ak-col>
                        <ak-col k-field="parmCategoryID" k-title="Parm Category"></ak-col>
                        <ak-col k-field="responseTypeCategoryID" k-title="Response Type"></ak-col>
                        <ak-col k-field="responseValue" k-title="Response Value"></ak-col>
                        <ak-col k-field="nextNodeID" k-title="Next Node"></ak-col>
                        <ak-col k-title="&nbsp;">
                            <ak-template>
                                <div>
                                    <button type="button" class="btn btn-danger" click.delegate="deleteResponse($this)">Delete</button>
                                </div>
                            </ak-template>
                        </ak-col>
                    </ak-grid>
```

```TS
    export class DecisionTreeNodeView {

    dataSource = new kendo.data.DataSource();
    constructor(){
        this.dataSource.data(new Array<DecisionNodeResponse>());
    }
    deleteResponse(currentResponse) {
        alert('asdfadsf');
    }
}

```

**Comment by JeroenVinke** [comment]
Hey @brandonseydel 

I do not see anything wrong in your code. Also, it seems to work in a [gist](https://gist.run/?id=7c937588db40d7ab5f0af2f532eb517b) so that is a bit strange.

**Comment by martyns** [comment]
Is it possible that the grid is within a repeat.for? I can recreate this in this [gist](https://gist.run/?id=3ed6a711c5cd2b678af41025d2c53346).

This a problem I am having, so that lead me here.

**Comment by brandonseydel** [comment]
I am not using it with either of those cases.  I will have to try and get a better example out.

**Comment by martyns** [comment]
@JeroenVinke should I open a separate bug for the repeat.for issue?

**Comment by JeroenVinke** [comment]
@martyns I have created https://github.com/aurelia-ui-toolkits/aurelia-kendoui-bridge/issues/685. We are planning on creating a new release in the next few days that will include a few new features and fixes, that one included.

**Comment by brandonseydel** [comment]
I wasn't even in a repeat.  I was on the latest version and after downgrading everything worked with no code change.

**Comment by JeroenVinke** [comment]
@brandonseydel the code you have provided seems to work for me (with the latest version of the bridge):

![image](https://cloud.githubusercontent.com/assets/2189477/21897061/c1772714-d8e8-11e6-84fe-bdc95728530a.png)

So something else seems to be going wrong. I have pushed the app I tested with [here](https://github.com/JeroenVinke/kendo-issue-682).

Did you perform an upgrade perhaps? If so, from what version did you upgrade?

**Comment by brandonseydel** [comment]
0.0.12.2 -> 1.0.1 Broke the events inside of the grid.

**Comment by JeroenVinke** [comment]
There were definitely some breaking changes between 0.12.2 and 1.0.1 but seeing that you have no issues displaying the grid I don't think that's it either. Let me know when you have reproduced the issue in an app that I can debug

**Comment by brandonseydel** [comment]
how about making another bundle for me to reference with rawgit.  I am at my day job and cannot gulp bundle and everything on my own fork atm

**Comment by JeroenVinke** [comment]
Sure, [here](https://gist.run/?id=023c72bbef47863e9d3b1c8ed9638279) is a sample with aurelia-kendoui-bridge@1.0.1 with a click event

**Comment by JeroenVinke** [comment]
Since the issue description is quite similar to the one in https://github.com/aurelia-ui-toolkits/aurelia-kendoui-bridge/issues/685, can you try and update your app to 1.1.1 and see if that fixes it? I have also created a bundle for 1.1.1 (you can change the bundle in index.html of any gist)

**Comment by JeroenVinke** [comment]
i'm unable to reproduce this problem, please reopen if this is still an issue
